### DISCUSSION TITLE
It is not possible to use a custom command with `@milkdown/plugin-listener`

### DISCUSSION THREAD

**Discussion by zgrybus** [Milkdown/milkdown#949] [Q&A]
### Initial checklist

- [X] I agree to follow the [code of conduct](https://github.com/Saul-Mirone/milkdown/blob/main/CODE_OF_CONDUCT.md)
- [X] I searched [issues](hhttps://github.com/Saul-Mirone/milkdown/issues) and [discussions](https://github.com/Saul-Mirone/milkdown/discussions) and couldn’t find anything (or linked relevant results below)

### Affected packages and versions

7.1.0

### Steps to reproduce

- Go to https://codesandbox.io/s/smoosh-sunset-ey4uhy?file=/src/EditorWithListener.tsx
- Highlight particular text
- Click on the `Transform to subscript`
- Error appers

( when you uncomment `ctx.get(listenerCtx).markdownUpdated((_, markdown) => {});` you will be able to transform text to subscript )

### Expected behavior

I am able to use a custom command when I use @milkdown/plugin-listener

### Actual behavior

When I want to use the custom command with `@milkdown/plugin-listener` I get the error 
> **Cannot handle unknown node `subscript'**

 
( subscript is a custom command )

### Runtime

Chrome, Safari, Electron

### OS

Windows, Linux, macOS

**Comment by Saul-Mirone** [comment]
It's not something about the listener plugin. The problem is that you don't handle the remark AST very well. You're trying to give the `subscript` as the name of the AST node but remark cannot recognize it. So if you try to transform your prosemirror state into markdown. It will throw the error. The proper way to do that is to add a remark plugin for it.

**Comment by zgrybus** [comment]
HI @Saul-Mirone 
So basically, I should write something similar to https://www.npmjs.com/package/remark-sub-super? 
and wrap it with `$remark` function? 🤔 

Unfortunately, this lib is obsolete..

**Reply by Saul-Mirone** [reply]
Yep. You can also use other syntax plugins such as https://github.com/remarkjs/remark-directive
